.travis.yml

```diff
@@ -1,12 +1,17 @@
 # WARNING:
 # Do not use this file as a template for you repository TravisCI configuration
 # Instead use the files under the "/sample_files" directory for that purpose.
-sudo: required
 language: python
 
 python:
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - expect-dev
```

Comment:
I would be great if we could ditch `unbuffer`. I don't recall exactly why using something like `tee` instead didn't cut it...